internal/compiler/passes/apply_default_properties_from_style.rs

```diff
@@ -30,13 +31,32 @@ pub fn apply_default_properties_from_style(
                             "text-cursor-width",
                         ))
                     });
-                    elem.set_binding_if_not_set("color".into(), || Expression::Cast {
-                        from: Expression::PropertyReference(NamedReference::new(
-                            &style_metrics.root_element,
-                            "default-text-color",
```

Comment:
Why replace the binding for `default-text-color` to `Palette.foreground` only for `TextInput` but not `Text`?

Along the same lines, wouldn't it make sense to do the same dance for `Window.background`?